master/getting-started/kubernetes/reqs/system.md

```diff
@@ -0,0 +1,39 @@
+---
+title: System requirements
+canonical_url: 'https://docs.projectcalico.org/v3.0/getting-started/kubernetes/reqs/system'
+---
+
+{% include {{page.version}}/reqs-sys.md orch="Kubernetes" %}
+
+## Kubernetes
+
+#### Supported versions
+
+{{site.prodname}} {{page.version}} requires Kubernetes 1.8 or later.
+
+#### CNI plug-in enabled
+
+Enable CNI by passing `--network-plugin=cni` to kubelet. On kubeadm, this is
+the default.
+  
+#### Supported kube-proxy flags
+
+Do not use the following flags:
+- `--masquerade-all`
+- `--proxy-mode=userspace`
+
+Like Kubernetes, {{site.prodname}} offers beta support for `--proxy-mode=ipvs`. 
```

Comment:
This paragraph is worded oddly - I wouldn't say "Like kubernetes, Calico offers beta support for proxy-mode=ipvs"

I'd recommend wording like this:


```
Calico supports the following kube-proxy modes:
- iptables
- ipvs (beta, requires Kubernetes >=v1.9.3)
```

This means we can also get rid of the `--proxy-mode=userspace` bit above because it's clearly not a supported proxy mode.